src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceComplaintsAsserts.java

```diff
@@ -0,0 +1,119 @@
+       assertTrue(complaintFormTitle.contains(title), "Expect " + title + " to be found in " + complaintFormTitle);
+    }
+
+    /**
+     * Verify that complaint was added successfully.
+     * Assert value in ONC-ACB Complaint Id column in complaints view table
+     * @param acbComplaintId is value to look for in Complaint Id column
+     */
+    @Then("^a new complaint with given \"([^\"]*)\" should be added to CHPL$")
+    public void verifyComplaintIsAdded(final String acbComplaintId) {
+        getDriver().navigate().refresh();
+        boolean isFound = false;
+
+        WebElement table = SurveillanceComplaintsPage.complaintsTable(getDriver());
+        List<WebElement> rows = table.findElements(By.xpath("tbody/tr"));
+
+        for (WebElement row : rows) {
+            List<WebElement> cols = row.findElements(By.xpath("td"));
+            if (cols.get(ACBCOMPLAINTID_COL).getText().equals(acbComplaintId)) {
+                isFound = true;
+                break;
+            }
+        }
+        assertTrue(isFound, "Text: " + acbComplaintId + " not found");
+    }
+
+    @Then("^\"([^\"]*)\" form should open$")
+    public void verifyEditComplaintFormOpens(final String title) {
+        String complaintFormTitle = SurveillanceComplaintsPage.addComplaintFormTitle(getDriver()).getText();
+        assertTrue(complaintFormTitle.contains(title), "Expect " + title + " to be found in " + complaintFormTitle);
+    }
+
+    /**
+     * Verify that edits to complaint data saved successfully.
+     * Assert value in ONC Complaint Id column in complaints view table post save
+     * @param oncComplaintID is value to look for in ONC Complaint Id column
+     */
+    @Then("^the edits should save and show \"([^\"]*)\" in complaints view table$")
+    public void verifyEditsSave(final String oncComplaintID) {
+        getDriver().navigate().refresh();
+        boolean isFound = false;
+
+        WebElement table = SurveillanceComplaintsPage.complaintsTable(getDriver());
+        List<WebElement> rows = table.findElements(By.xpath("tbody/tr"));
+
+        for (WebElement row : rows) {
+            List<WebElement> cols = row.findElements(By.xpath("td"));
+            if (cols.get(ONCCOMPLAINTID_COL).getText().equals(oncComplaintID)) {
+                isFound = true;
+                break;
+            }
+        }
+        assertTrue(isFound, "Text: " + oncComplaintID + " not found");
+    }
+
+    /**
+     * Verify that complaint was deleted successfully.
+     * Assert that value is not found in ONC-ACB Complaint Id column in complaints view table
+     * @param acbComplaintId is value to look for in Complaint Id column
+     */
+    @Then("^the complaint with ONC-ACB Complaint ID \"([^\"]*)\" should not display in complaints view table$")
+    public void verifyComplaintIsDeleted(final String acbComplaintId) {
+        getDriver().navigate().refresh();
+        boolean isFound = true;
```

Comment:
`isFound` should start as false, and switch to true when it's found. "Is found" being false when the text was found is backwards

Reply:
Updated test to use assertFalse